traits/tests/test_view_elements.py

```diff
@@ -125,3 +125,8 @@ class Model(HasTraits):
 
         with self.assertRaises(TraitError):
             Model.class_trait_view_elements()
+
+    def test_view_element_superclass(self):
+        from traitsui.api import ViewElement
+
+        self.assertIsInstance(ViewElement(), AbstractViewElement)
```

Comment:
Possible overkill, but it might be worth testing for `issubclass(ViewElement, AbstractViewElement)` as well.  That rules out `ViewElement` becoming a factory function at some future point, but I think that's OK.

Reply:
Well, in the long run, we don't want this test class here at all. The TraitsUI test suite is the right place to check that `ViewElement` behaves appropriately.

Reply:
Is it useful to enforce this limitation on `ViewElement`? From what I see, Traits always uses  `isinstance` check, so it should work perfectly fine even if `ViewElement` becomes a factory function. Or am I missing something?

Reply:
Agreed: all Traits cares about is that the view is an instance of `AbstractViewElement`, so I think it's enough to check just that here.

And again, really this check should be in TraitsUI: if there isn't a test for this in TraitsUI, we should add one. (And once it's there, we can remove this one.)